### PR TITLE
Create dm_coordination.md

### PR DESCRIPTION
**Pull request by floradanna**
Coordination of DM in collaborative projects

### CODE REVIEW

File: pages/your_tasks/dm_coordination.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* Automate backups as much as possible with your IT department and test them. Backup log files should be actively monitored for potential issues.
+* The DMWG should perform regular checks with data hosting project partners to check that data and metadata are produced according to the DMP, and that agreed backup routines are followed. 
+* Post-project preservation of data should consider reproducibility/replicability of the data from a long-term perspective, as well as the potential reuse value. You might want to consult a [data preservation](preserving) service (e.g. national data centres) regarding larger amounts of data. As an initial inspiration, you can start from [this data value checklist](https://www.bgs.ac.uk/download/ngdc-data-value-checklist/) and adapt it for your legal framework and domain.
+* Do not leave the data holders with an impossible task of trying to figure out who can have access to the resources, but rather use a license that allows re-use of data - e.g. some projects (project call level) demand to use CC-BY-4.0 and refrain from using bespoke, non-standard license terms.
```
Comment (Nazeeefa) [inline pages/your_tasks/dm_coordination.md:88]: what does (project call level) refer to? 
or did you mean to say "(depending on activity level)"?

**Review by Nazeeefa — CHANGES_REQUESTED** [review]

File: pages/your_tasks/dm_coordination.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+title: Project data management coordination
+description: how to coordinate and organise data management activities in collaborative or multi-parter projects
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md:3]: multi-partner

File: pages/your_tasks/dm_coordination.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+## How to organize data management in collaborative projects?
+ 
+### Description
+
+Most national and European funders of (multiparty) projects now require [Data management plans (DMPs)](data_management_plan) that are evaluated before grant awards. Therefore, coordinators of project consortia need to mobilise a dedicated effort to decide on the approach for data management, developing the DMP, using the DMP as a live document over the course of  the project, etc, to ensure that the research outputs from the project follow the [FAIR principles](about#why-are-the-fair-principles-needed). This is new terrain for many people, requiring them to grasp what this entails and figure out who to approach.
+
+### Considerations
+
+* During the writing stage of a grant proposal, members of a multi partners project or consortium need to agree on how to address data management during (and after) the project, and formalise the way of working in the proposal.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md:22]: The partners have to first agree on how to address DM during and after the project, and then they have to write it in the proposal.

File: pages/your_tasks/dm_coordination.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+* What information regarding data management is needed at different phases (preparation, submission, execution, wrap up, reporting to the funders) in the project needs to be decided in advance.
+* There is a need to understand the institutional landscape in terms of legal requirements, data policies, IT services, etc. that are needed to carry out the project. They might vary among individual institutions in the project consortium:
+  * Not every institution has a centralised IT department to support data management; often data management is carried out by the group itself without help from the IT department.
+  * Legal and ethical requirements, data policies, etc. can be very different or even not be in place in some institutions.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md:26]: I see your point, but I think that in this case a little bit of redundancy and very practical examples help the readers understanding what we are talking about.

File: pages/your_tasks/dm_coordination.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+* If applicable, the establishment of the DMWG can be included as a project task, deliverable or milestone. Each partner should allocate personnel time for the DMWG. Dedicating part of the project (a work package, for instance) to data management helps with clarifying tasks and responsibilities. The DMWG should have defined milestones and deliverables in the project plan.
+* All partners should agree on what needs to be added to the budget for data management purposes: hardware requirements (long-term storage space, servers, etc.), cloud services, software licensing, software developers, and data stewards/managers (costs of making data FAIR). It is advisable to consult with available IT support, data centers, and instrument providers at each partner’s side to evaluate options and costs.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md:36]: yes, indeed. I will add your suggestion

File: pages/your_tasks/dm_coordination.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
+---
+title: Project data management coordination
+description: how to coordinate and organise data management activities in collaborative or multi-parter projects
+contributors: [Robert Andrews, Stefanie Meyer, Tereza Motalova, Graham Parton, Marko Petek, Maja Zagorščak, Karolina Zavoralova, Karel Berka, Korbinian Bösl, Flora D’Anna, Niclas Jareborg, Yvonne Kallberg, Paulette Lieby]
+page_id: DM coordination
+related_pages: 
+  tool_assembly: [<!---REPLACE THIS with the page ID of the tool_assembly pages that you want to list here as related pages--->]
```
Comment (bedroesb) [inline pages/your_tasks/dm_coordination.md:7]: small reminder :)

File: pages/your_tasks/dm_coordination.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+* Possible problems with data exchange between partners can resurface during the project. 
+* Consider contacting and establishing collaborations for depositing data with key repositories, e.g. setting up an ENA Compare Data Hub for depositing sequence data at the European Nucleotide Archive.
+
+### Solutions 
+* The DMWG should have regular meetings to find appropriate solutions to arising data management issues.
+* The DMWG should make sure all partners are able to implement the data management strategies.
+* Establish a strategy for documentation and exchange of data (within the project and with other collaborators):
+  * Evaluate the value of your project outcomes (datasets, data, tools, etc) and establish which of them are worth preserving long term. 
+  * Define the strategy as early as possible, preferably at the beginning of the project, otherwise it may generate problems.  
+  * Agree on data formats (e.g. TXT, MP4, FASTQ), metadata standards (e.g. MIAPPE, MIASE - depends on the type of data generated), and controlled vocabularies/ontologies.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md]: OK

File: pages/your_tasks/dm_coordination.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+* Ensure that all partners deposit data according to the agreed data publication strategy.
+* The DMP should be revisited on a regular basis and updated when necessary. Things to look out for are:
+  * Ensuring that the chosen [data repositories](data_publication) are fit for purpose.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md:64]: ok

File: pages/your_tasks/dm_coordination.md (hunk 1 above)
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md:88]: I think it means that some funder or specific type of project require CC-BY-4.0 licence.

File: pages/your_tasks/dm_coordination.md (hunk 6 above)
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md:7]: I think there are no related Tool Assembly

**Comment by floradanna** [comment]
@Nazeeefa thank you for your comments; I tried to address them. Please, approve this PR if you are happy with the changes so that it can be merged, or let me know if additional changes are needed.

File: pages/your_tasks/dm_coordination.md (hunk 6 above)
Comment (bedroesb) [inline pages/your_tasks/dm_coordination.md:7]:
```suggestion
  tool_assembly: []
```

File: pages/your_tasks/dm_coordination.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+  * Not every institution has a centralised IT department to support data management; often data management is carried out by the group itself without help from the IT department.
+  * Legal and ethical requirements, data policies, etc. can be very different or even not be in place in some institutions.
+* Data management strategies and the types of data generated can change during the project.
+
+
+### Solutions
+
+Here, we provide some advice and methods to help consortia with data management coordination tasks.
+
+* In a consortium, establishing a Data Management Working Group (DMWG) should be one of the first actions done during the establishment of the consortium. Each partner should have a representative for data management, especially the partners that will generate and/or use a higher volume of data. If possible, the representatives should have a working knowledge of data management. During the writing stage, the DMWG should be responsible for deciding how data management will be addressed during (and after) the project.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md]: Changed to "representatives".

I don't think we have a useful link to define working knowledge.  It probably means just "working experience". I would keep it as it is or replace with "experience" or "working experience". Let me know what you prefer.

File: pages/your_tasks/dm_coordination.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+* If applicable, the establishment of the DMWG can be included as a project task, deliverable or milestone. Each partner should allocate personnel time for the DMWG. Dedicating part of the project (a work package, for instance) to data management helps with clarifying tasks and responsibilities. The DMWG should have defined milestones and deliverables in the project plan.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_tasks/dm_coordination.md]: done. changed.

**Review by smza — APPROVED** [review]

File: pages/your_tasks/dm_coordination.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+* The DMWG should have regular meetings to find appropriate solutions to arising data management issues.
+* The DMWG should make sure all partners are able to implement the data management strategies.
+* Establish a strategy for documentation and exchange of data (within the project and with other collaborators):
+  * Evaluate the value of your project outcomes (datasets, data, tools, etc) and establish which of them are worth preserving long term. 
+  * Define the strategy as early as possible, preferably at the beginning of the project, otherwise it may generate problems.  
+  * Agree on data formats (e.g. TXT, MP4, FASTQ), metadata standards (e.g. MIAPPE, MIASE - depends on the type of data generated), and controlled vocabularies/ontologies.
+* Ensure that all partners deposit data according to the agreed data publication strategy.
+* The DMP should be revisited on a regular basis and updated when necessary. Things to look out for are:
+  * Ensuring that the chosen [data repositories](data_publication) are fit for purpose.
+  * Ensuring that the metadata standards and the ontologies used are fit for purpose.
```
[the remark this answers is not in the document]
Comment (nicjar) [reply pages/your_tasks/dm_coordination.md:65]: I agree (with keeping it)

**Review by Nazeeefa — APPROVED** [review]
Thanks @floradanna for addressing reviews and, updated version

**Pull request merged**